### PR TITLE
Examples validation

### LINKED ISSUE #262: Filter example submission
In the example submission form, there is placeholder text that explain some tricks on how to format your example.
Some people have been submitting examples containing nothing more than this example text.

We should add a check that alert the user if the text is the same than the example to avoid having crap in the database

### PR DESCRIPTION
**Pull request by ludov04**
Fixes #262

### CODE REVIEW

File: views/topic/show.ejs [hunk 1]
```diff
@@ -88,6 +88,17 @@
       </section>
     <% } %>
 
+    <% if(topic.aliases) { %>
```
Comment (filipsch) [inline views/topic/show.ejs]: This seems to be unrelated?

**Comment by ludov04** [comment]
It's to get all the aliases of the function in the document, so then in the javascript, I can check if at least one of them was used
